libs/financial-aid/shared/src/lib/taxCalculator.ts

```diff
@@ -203,24 +203,24 @@ export const acceptedAmountBreakDown = (amount?: Amount): Calculations[] => {
       }
     }) ?? []
 
-  const childrenAid =
-    amount?.childrenAidAmount && amount?.childrenAidAmount !== 0
+  const checkAid = (title: string, aid?: number) => {
+    return aid && aid !== 0
```

Comment:
Is this zero check needed here?

0 is falsy.... right?

Reply:
It is needed since when a applicant does not have children the children amount will be marked 0 and same with december 😇